### DISCUSSION TITLE
App doesn‘t load pictures in full size when previews haven‘t been created

### DISCUSSION THREAD

**Discussion by dariosun** [nextcloud/all-in-one#4186] [Questions] [answered]
Hey, I am just wondering about a behaviour that I find strange and I wondered if this is expected:
I am using the newest NC AIO images on docker behind a reverse proxy. 
When I look at my automatically uploaded pictures through the Nextcloud app by opening them full screen and then swiping, only those pictures show for which the previews have already been loaded after opening the respective folder. So for example, the first pictures work fine, but once I swipe „lower“, the pictures remain blank. However, exiting full screen, scrolling down and waiting for the previews to be loaded solves this.

On the website, I don‘t have this problem. Any solutions for this?

**Comment by szaimen** [comment]
> Nextcloud app

Which app? Android or ios?

**Reply by dariosun** [reply]
iOS

**Reply by szaimen** [reply, marked as answer]
This sounds like a bug in thr ios app then. Feel free to report this to https://github.com/nextcloud/ios

**Reply by dariosun** [reply]
Thank you!

**Comment by Legenda374** [comment]
I have the same problem, is there a solution to the problem? Or others issues for ios app?

**Reply by Legenda374** [reply]
I created issue https://github.com/nextcloud/ios/issues/2801
